internal/config/config.go

```diff
@@ -77,6 +83,16 @@ func initConfig() {
 		viper.SetDefault("DBPort", c.Database.Port)
 		viper.SetDefault("DBssl", c.Database.SslMode)
 
+		// clowder RBAC Config
+		for _, endpoint := range c.Endpoints {
+			if endpoint.App == "rbac" {
+				viper.SetDefault("RBACHost", endpoint.Hostname)
+				viper.SetDefault("RBACPort", endpoint.Port)
+				viper.SetDefault("RBACProtocol", "https")
```

Comment:
For any external communication, please go through this ticket - https://issues.redhat.com/browse/RHINENG-103. Please make sure you are using correct configurations from clowder.

Reply:
Yes we have taken care of that. For golang application we just need to add [SSL_CERT_DIR](https://github.com/RedHatInsights/ros-ocp-backend/blob/main/clowdapp.yaml#L210) environment variable.